packages/frontend/src/creation-form/index.tsx

```diff
@@ -28,20 +41,48 @@ export const Component = ({
         Specification["payload"] | undefined
     >(state.specification?.payload);
 
+    // TODO: properly implement minimum elapsed time enforcement
```

Comment:
Is this TODO still valid?

Reply:
Nope, addressed in e4fc01299b44e5e35b9b90ddd1967e83efb6cb85